src/gh_pr/core/batch.py

```
@@ -1,423 +1,436 @@
+                for owner, repo, pr_number in pr_identifiers:
+                    future = executor.submit(self._execute_single_operation, operation, owner, repo, pr_number)
+                    future_to_pr[future] = (owner, repo, pr_number)
+
+                # Process completed futures
+                for future in as_completed(future_to_pr):
+                    owner, repo, pr_number = future_to_pr[future]
+
+                    try:
+                        result = future.result()
+                        results.append(result)
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)